pkg/metadata.go

```diff
@@ -25,7 +25,6 @@ type NlxMetadata struct {
 	PermissionSets     []string `json:"permissionsets"`
 	SitePermissionSets []string `json:"sitepermissionsets"`
 	SessionVariables   []string `json:"sessionvariables"`
-	Profiles           []string `json:"profiles"`
```

Comment:
I don't think we should need this anymore since we have `SitePermissionSets` in the struct

Reply:
It was in there for backwards compatibility with "older" pliny, which isn't deployed anywhere anymore, so... yep, we can delete it. Esp. for CLI 0.6.x